.golangci.yml

```diff
@@ -0,0 +1,38 @@
+run:
+  timeout: 5m
+  tests: false
```

Comment:
@jasonpaulos Since avm-abi is a smaller repo than go-algorand, is there a reason _not_ to check the tests?

```suggestion

```

Reply:
Visual reminder:  If a change is made here, then it also ought to be made in `-warnings.yml`.

Reply:
Thanks for pointing this out. I'm unsure why tests were disabled in go-algorand, but I've opted to enable them in 2e0c191 for both the standard and `-warnings.yml` configurations.

If we find the linter is too strict on test code going forward, we can adjust as needed.

Reply:
@jasonpaulos Sounds good - Thanks for trialing the change.  Resolving.